<!-- -*- mode: XML -*- -->

<launch>
    <node name="dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam>
            namespace: dxl_manager
            serial_ports:
                ttlUSB0_port:
                    port_name: "/dev/ttyUSB0"
                    baud_rate: 500000
                    min_motor_id: 0
                    max_motor_id: 5
                    update_rate: 10
        </rosparam>
    </node>
</launch>
